src/aspire/abinitio/commonline_base.py

```diff
@@ -17,7 +17,14 @@ class CLOrient3D:
     """
 
     def __init__(
-        self, src, n_rad=None, n_theta=360, n_check=None, max_shift=0.15, shift_step=1
+        self,
+        src,
+        n_rad=None,
+        n_theta=360,
+        n_check=None,
+        max_shift=0.15,
+        shift_step=1,
+        mask=True,
```

Comment:
Yeah, I wasn't sure. I don't know if really needs to be super flexible. It may just be that it's better to use the mask in some cases (ie. noisy images) and not in others (ie. clean images, class averages ?). So a simple on/off button might be enough.

Either way, I'm open to discussion on it. 

FYI, this mask is applied to the images in real space.

Reply:
Okay, let's see if Joakim wants to bring it up.  Otherwise it is fine by me for now. Thanks!